Makefile.js

```diff
@@ -680,6 +687,8 @@ target.gensite = function(prereleaseVersion) {
             let text = cat(filename),
                 title;
 
+            process.stdout.write(`> Updating files (Steps 4-9): ${i}/${length} - ${sourcePath + " ".repeat(30)}\r`);
```

Comment:
It resets to the beginning of the line, instead of spamming the terminal with lots and lots of lines.